person.rb

```diff
@@ -1,14 +1,19 @@
 require './nameable'
+require './capitalize_decorator'
+require './trimmer_decorator'
+require './rental'
 
 class Person < Nameable
-  attr_accessor :id, :name, :age
+  attr_accessor :name, :age
```

Comment:
- [ ] You might have forgotten to add the rental to the attribute accessor. Kindly add it.